packages/webcert/src/feature/certificate/Question/Question.tsx

```diff
@@ -89,48 +89,82 @@ const Question: React.FC<QuestionProps> = ({ id, className }) => {
+        break
       case ConfigTypes.UE_DROPDOWN:
-        return <UeDropdown {...commonProps} />
+        component = <UeDropdown {...commonProps} />
+        break
       case ConfigTypes.UE_RADIO_MULTIPLE_CODE:
-        return <UeRadioGroup {...commonProps} />
+        component = <UeRadioGroup {...commonProps} />
+        break
       case ConfigTypes.UE_CHECKBOX_MULTIPLE_DATE:
-        return <UeCheckboxDateGroup {...commonProps} />
+        component = <UeCheckboxDateGroup {...commonProps} />
+        break
       case ConfigTypes.UE_SICK_LEAVE_PERIOD:
-        return <UeSickLeavePeriod {...commonProps} />
+        component = <UeSickLeavePeriod {...commonProps} />
+        break
       case ConfigTypes.UE_DIAGNOSES:
-        return <UeDiagnoses {...commonProps} />
+        component = <UeDiagnoses {...commonProps} />
+        break
       case ConfigTypes.UE_RADIO_MULTIPLE_CODE_OPTIONAL_DROPDOWN:
-        return <UeRadioGroupOptionalDropdown {...commonProps} />
+        component = <UeRadioGroupOptionalDropdown {...commonProps} />
+        break
       case ConfigTypes.UE_UNCERTAIN_DATE:
-        return <UeUncertainDate {...commonProps} />
+        component = <UeUncertainDate {...commonProps} />
+        break
       case ConfigTypes.UE_MESSAGE:
-        return <UeMessage {...commonProps} />
+        component = <UeMessage {...commonProps} />
+        break
       case ConfigTypes.UE_TYPE_AHEAD:
-        return <UeTypeahead {...commonProps} />
+        component = <UeTypeahead {...commonProps} />
+        break
       case ConfigTypes.UE_TEXTFIELD:
-        return <UeTextField {...commonProps} />
+        component = <UeTextField {...commonProps} />
+        break
       case ConfigTypes.UE_DATE:
-        return <UeDate {...commonProps} />
+        component = <UeDate {...commonProps} />
+        break
       case ConfigTypes.UE_CAUSE_OF_DEATH:
-        return <UeCauseOfDeath {...commonProps} />
+        component = <UeCauseOfDeath {...commonProps} />
+        break
       case ConfigTypes.UE_CAUSE_OF_DEATH_LIST:
-        return <UeCauseOfDeathList {...commonProps} />
+        component = <UeCauseOfDeathList {...commonProps} />
+        break
       case ConfigTypes.UE_HEADER:
-        return
+        break
       default:
-        return <InfoBox type="error">Cannot find a component for: {question.config.type}</InfoBox>
+        component = <InfoBox type="error">Cannot find a component for: {question.config.type}</InfoBox>
+    }
+    if (!question.config.accordion) {
+      return component
+    } else {
+      return (
+        <Accordion
+          icon={question.config.icon}
+          includeIconTooltip
+          titleId={question.id}
+          title={question.config.accordion.openText}
+          titleClose={question.config.accordion.closeText}
+          header={question.config.accordion.header}
+          isControl={true}>
+          {component}
+        </Accordion>
+      )
```

Comment:
Instead of assigning to `component` we can just wrap this function.

```typescript
if (question.config.accordion) {
  return <Accordion ...>getUnifiedEditComponent(question, disabled)</Accordion>
}
return getUnifiedEditComponent(question, disabled)
```